src/webapp/pages/form-page/incident-action/mapIncidentActionToInitialFormState.ts

```diff
@@ -381,23 +386,36 @@ function getResponseActionSection(options: {
     return responseActionSection;
 }
 
-export function addNewResponseActionSection(sections: FormSectionState[]): FormSectionState {
+export function addNewResponseActionSection(
+    sections: FormSectionState[],
+    isIncidentManager: boolean
+): FormSectionState {
     const responseActionSections = sections.filter(
         section => !section.id.startsWith("addNewResponseActionSection")
     );
 
+    const searchAssignROField = getSectionsField(sections, responseActionConstants.searchAssignRO);
+    const statusField = getSectionsField(sections, responseActionConstants.status);
+    const verificationField = getSectionsField(sections, responseActionConstants.verification);
+
     const newResponseActionSection = getResponseActionSection({
         incidentResponseAction: undefined,
         options: {
             searchAssignROOptions:
-                sections[0]?.fields[3]?.type === "select" ? sections[0].fields[3].options : [],
-            statusOptions:
-                sections[0]?.fields[6]?.type === "select" ? sections[0].fields[6].options : [],
+                searchAssignROField?.type === "select" ? searchAssignROField.options : [],
```

Comment:
@deeonwuli now we have configurations in useAppContext which will give you the options for each form type. Please use that instead : configurations.selectableOptions.incidentActionPlanConfigurations